frontend/apps/filemanager/filemanagerfilesearcher.lua

```diff
@@ -122,6 +122,16 @@ function FileSearcher:close()
     end
 end
 
+local help_text = [[If enabled and the search pattern contains '*' or '?' these will be used as wildcards.
+
+'c*.jpg' will find all files starting with 'c' and end with '.jpg' (e.g. 'cover.jpg').
+
+
+If disabled the search pattern will be interpreted as a lua expression.
+
+'c*.jpg' will find all files starting with zero or more 'c' and end with '.jpg' (e.g. 'cccc.jpg').
+]]
```

Comment:
(It's always `Lua` with an uppercase L.)
I'd just use standard wildcards by default. Nobody should need to use Lua expressions - and we shouldn't tell users the code is in that obscure language called Lua that nobody knows (unless there is no alternative and it's good to know that - but you just added an alternative :)

Reply:
Shall I let the checkbox there, just without the help text (and enabled by default)?

Reply:
I'd say just remove it and the global setting, and do your substitutions always.
(* and ? can't be in filename, so if they are input'ed, their meaning is obviously wildcard.)

A description (or at least a help_text in the menu item) saying that one can use ? and * as wildcard could be nice, but not needed.
But a last_search like you did for fulltext search would be nice :) so we don't have to re-enter the whole thing in case of no match.

Reply:
> A description (or at least a help_text in the menu item) saying that one can use ? and * as wildcard could be nice, but not needed.

In the filemangermenu.lua there are the Lua-patterns already mentioned. So the checkbox with wildcards would be no additional alternative.
How should I go on: 
1.) Drop the checkbox, remove the lua-patterns mentioned in filemanagermenu and add wildcards information there?
2.) Let it be as it is (checkbox, help_text)?

> But a last_search like you did for fulltext search would be nice :) so we don't have to re-enter the whole thing in case of no match.

That is already in there. As long as you stay in FM the search pattern is stored. Shall I store it permanently in `settings.reader.lua`?

Reply:
> 1.) Drop the checkbox, remove the lua-patterns mentioned in filemanagermenu and add wildcards information there?

Yep, I think this is better.

> That is already in there. As long as you stay in FM the search pattern is stored. Shall I store it permanently in settings.reader.lua?

Oh ok, good, No need to have it stored in settings.reader.lua.